tests/Nhs.Appointments.Api.Integration/BaseFeatureStepsTests.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using FluentAssertions;
+using Nhs.Appointments.Api.Integration.Scenarios;
+using Xunit;
+
+namespace Nhs.Appointments.Api.Integration;
+
+public class BaseFeatureStepsTests
+{
+    [Theory]
+    [InlineData("Today", 0)]
+    [InlineData("Tomorrow", 1)]
+    [InlineData("Yesterday", -1)]
+    [InlineData("3 days from now", 3)]
+    [InlineData("3 days from today", 3)]
+    [InlineData("3 days before now", -3)]
+    [InlineData("1 day before now", -1)]
+    [InlineData("1 week from now", 7)]
+    [InlineData("2 weeks from today", 14)]
+    [InlineData("1 month before now", -28)]
```

Comment:
Is this correct? I expect one month before now to be the 20th October which is 31 days